dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/AbstractClouderaManagerTask.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.cloudera.manager;
+
+import com.google.common.io.ByteSink;
+import com.google.edwmigration.dumper.application.dumper.handle.Handle;
+import com.google.edwmigration.dumper.application.dumper.task.AbstractTask;
+import com.google.edwmigration.dumper.application.dumper.task.TaskRunContext;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+public abstract class AbstractClouderaManagerTask extends AbstractTask<Void> {
```

Comment:
Naming a bit confusing for this class. `ClouderaClustersTask` also extends it, but does not have `manager` suffix. Can it be renamed to `AbstractClouderaTask`?

Reply:
Well, you are absolutely right, the current naming with Cloudera is confusing. 
We have `cloudera` connector which works just with Hadoop. Fix this confusion is planned in the next PR to keep the scope of PRs smaller. 

The scope of the current task is to add connector which will work with Cloudera Manager API and will be irrelevant for any other Hadoop vendors. 
So, this abstract task is parent for each specific task which works with Cloudera Manager API, kind of grouping and some utility help. 

Current naming:
For example `ClouderaClustersTask` dump data related to Cloudera's `Hadoop clusters. It's done by Cloudera Manager API, but still it's it's Cloudera's Hadoop clusters, not Cloudera Manager clusters. 

I'm open to any ideas to fix the confusion and improve the naming, please share, I will consider the ideas and try to fix the issue in the next PR.